apps/web/src/components/Basenames/PoweredByEns/index.tsx

```diff
@@ -0,0 +1,80 @@
+import Image, { StaticImageData } from 'next/image';
+
+import base from './assets/base.svg';
+import ens from './assets/ens.svg';
+import classNames from 'classnames';
+
+enum CircleColors {
+    Green='green',
+    Blue='blue',
+    Pink='pink',
+    Grey='gray',
+}
+
+type CircleProps = {
+    color: string;
+    size: string;
+    maxWidth: string;
+    className: string;
+}
+
+function Circle({ color, size, maxWidth, className }: CircleProps) {
+    // bg-grey-40, bg-pink-15, bg-green-15, bg-blue-15
+    // w-[14%], w-[8%], w-[20%]
+    // max-w-[71.25], max-w-[41.84], max-w-[102.51], max-w-[70.11]
+    // pt-[14%], pt-[8%], pt-[20%]
```

Comment:
```suggestion
```

Reply:
These comments allow tailwind to dynamically interpolate styles, in order for it to work correctly the full value needs to be somewhere in the file.

Reply:
I could change them to be inline styles though per: https://v2.tailwindcss.com/docs/just-in-time-mode

Reply:
oh interesting I had no idea this was a thing

Reply:
If someone changes the value in `<Circle ...` without updating the commented bits, it'll break.

I'd just pass all these classes in `Circle  className="...`, it's fine if there's some repetition.